Building on Windows is failing

Same error reproduced on 2 different machines:

```
(.venv) C:\Users\ben.hardcastle>python -m pip install wavpack-numcodecs
Looking in indexes: https://pypi.org/simple, http://aibspi:3141
Collecting wavpack-numcodecs
  Downloading wavpack-numcodecs-0.2.1.tar.gz (2.6 MB)
     ---------------------------------------- 2.6/2.6 MB 15.0 MB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Requirement already satisfied: numpy in c:\users\ben.hardcastle\.venv\lib\site-packages (from wavpack-numcodecs) (1.24.4)
Requirement already satisfied: packaging in c:\users\ben.hardcastle\.venv\lib\site-packages (from wavpack-numcodecs) (24.1)
Collecting numcodecs
  Downloading numcodecs-0.12.1-cp38-cp38-win_amd64.whl (793 kB)
     ---------------------------------------- 793.9/793.9 kB 10.0 MB/s eta 0:00:00
Building wheels for collected packages: wavpack-numcodecs
  Building wheel for wavpack-numcodecs (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for wavpack-numcodecs (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [225 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build\lib.win-amd64-cpython-38\wavpack_numcodecs
      copying src\wavpack_numcodecs\globals.py -> build\lib.win-amd64-cpython-38\wavpack_numcodecs
      copying src\wavpack_numcodecs\__init__.py -> build\lib.win-amd64-cpython-38\wavpack_numcodecs
      running egg_info
      writing src\wavpack_numcodecs.egg-info\PKG-INFO
      writing dependency_links to src\wavpack_numcodecs.egg-info\dependency_links.txt
      writing entry points to src\wavpack_numcodecs.egg-info\entry_points.txt
      writing requirements to src\wavpack_numcodecs.egg-info\requires.txt
      writing top-level names to src\wavpack_numcodecs.egg-info\top_level.txt
      reading manifest file 'src\wavpack_numcodecs.egg-info\SOURCES.txt'
      reading manifest template 'MANIFEST.in'
      C:\Users\ben.hardcastle\AppData\Local\Temp\pip-build-env-ndx4tt6e\overlay\Lib\site-packages\setuptools\config\_apply_pyprojecttoml.py:71: _MissingDynamic: `entry-points` defined outside of `pyproject.toml` is ignored.
      !!

              ********************************************************************************
              The following seems to be defined outside of `pyproject.toml`:

              `entry-points = {'numcodecs.codecs': ['wavpack = wavpack_numcodecs:WavPack']}`

              According to the spec (see the link below), however, setuptools CANNOT
              consider this value unless `entry-points` is listed as `dynamic`.

              https://packaging.python.org/en/latest/specifications/pyproject-toml/#declaring-project-metadata-the-project-table

              To prevent this problem, you can list `entry-points` under `dynamic` or alternatively
              remove the `[project]` table from your file and rely entirely on other means of
              configuration.
              ********************************************************************************

      !!
        _handle_missing_dynamic(dist, project_table)
      warning: no files found matching '*.c' under directory 'wavpack_numcodecs'
      warning: no files found matching '*.pyx' under directory 'wavpack_numcodecs'
      warning: no files found matching '*.pxd' under directory 'wavpack_numcodecs'
      warning: no files found matching '*.lib' under directory 'wavpack_numcodecs'
      warning: no files found matching '*.dll' under directory 'wavpack_numcodecs'
      warning: no files found matching '*.h' under directory 'wavpack_numcodecs\include'
      warning: no files found matching '*' under directory 'wavpack_numcodecs\libraries'
      adding license file 'LICENSE'
      writing manifest file 'src\wavpack_numcodecs.egg-info\SOURCES.txt'
      copying src\wavpack_numcodecs\compat_ext.c -> build\lib.win-amd64-cpython-38\wavpack_numcodecs
      copying src\wavpack_numcodecs\compat_ext.pxd -> build\lib.win-amd64-cpython-38\wavpack_numcodecs
      copying src\wavpack_numcodecs\compat_ext.pyx -> build\lib.win-amd64-cpython-38\wavpack_numcodecs
      copying src\wavpack_numcodecs\libwavpack.so.1 -> build\lib.win-amd64-cpython-38\wavpack_numcodecs
      copying src\wavpack_numcodecs\wavpack.c -> build\lib.win-amd64-cpython-38\wavpack_numcodecs
      copying src\wavpack_numcodecs\wavpack.pyx -> build\lib.win-amd64-cpython-38\wavpack_numcodecs
      copying src\wavpack_numcodecs\wavpackdll.dll -> build\lib.win-amd64-cpython-38\wavpack_numcodecs
      creating build\lib.win-amd64-cpython-38\wavpack_numcodecs\include\wavpack
      copying src\wavpack_numcodecs\include\wavpack\wavpack.h -> build\lib.win-amd64-cpython-38\wavpack_numcodecs\include\wavpack
      creating build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.6.0\linux-x86_64-glibc2.35
      copying src\wavpack_numcodecs\libraries\5.6.0\linux-x86_64-glibc2.35\libwavpack.so -> build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.6.0\linux-x86_64-glibc2.35
      creating build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.6.0\windows-x86_32
      copying src\wavpack_numcodecs\libraries\5.6.0\windows-x86_32\wavpackdll.dll -> build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.6.0\windows-x86_32
      creating build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.6.0\windows-x86_64
      copying src\wavpack_numcodecs\libraries\5.6.0\windows-x86_64\wavpackdll.dll -> build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.6.0\windows-x86_64
      creating build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.7.0\linux-x86_64-glibc2.31
      copying src\wavpack_numcodecs\libraries\5.7.0\linux-x86_64-glibc2.31\libwavpack.so -> build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.7.0\linux-x86_64-glibc2.31
      creating build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.7.0\linux-x86_64-glibc2.35
      copying src\wavpack_numcodecs\libraries\5.7.0\linux-x86_64-glibc2.35\libwavpack.so -> build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.7.0\linux-x86_64-glibc2.35
      copying src\wavpack_numcodecs\libraries\5.7.0\linux-x86_64-glibc2.35\libwavpack.so.1 -> build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.7.0\linux-x86_64-glibc2.35
      creating build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.7.0\linux-x86_64-glibc2.39
      copying src\wavpack_numcodecs\libraries\5.7.0\linux-x86_64-glibc2.39\libwavpack.so -> build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.7.0\linux-x86_64-glibc2.39
      creating build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.7.0\windows-x86_32
      copying src\wavpack_numcodecs\libraries\5.7.0\windows-x86_32\wavpackdll.dll -> build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.7.0\windows-x86_32
      creating build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.7.0\windows-x86_64
      copying src\wavpack_numcodecs\libraries\5.7.0\windows-x86_64\wavpackdll.dll -> build\lib.win-amd64-cpython-38\wavpack_numcodecs\libraries\5.7.0\windows-x86_64
      creating build\lib.win-amd64-cpython-38\wavpack_numcodecs\src
      copying src\wavpack_numcodecs\src\decoder.c -> build\lib.win-amd64-cpython-38\wavpack_numcodecs\src
      copying src\wavpack_numcodecs\src\encoder.c -> build\lib.win-amd64-cpython-38\wavpack_numcodecs\src
      running build_ext
      building 'wavpack_numcodecs.compat_ext' extension
      creating build\temp.win-amd64-cpython-38\Release\src\wavpack_numcodecs
      "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -IC:\Users\ben.hardcastle\.venv\include -I\\allen\programs\mindscope\workgroups\dynamicrouting\ben\envs\win\np_pipeline_qc\include -I\\allen\programs\mindscope\workgroups\dynamicrouting\ben\envs\win\np_pipeline_qc\Include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /Tcsrc/wavpack_numcodecs/compat_ext.c /Fobuild\temp.win-amd64-cpython-38\Release\src/wavpack_numcodecs/compat_ext.obj
      compat_ext.c
      "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\bin\HostX86\x64\link.exe" /nologo /INCREMENTAL:NO /LTCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:C:\Users\ben.hardcastle\.venv\libs /LIBPATH:\\allen\programs\mindscope\workgroups\dynamicrouting\ben\envs\win\np_pipeline_qc\libs /LIBPATH:\\allen\programs\mindscope\workgroups\dynamicrouting\ben\envs\win\np_pipeline_qc /LIBPATH:C:\Users\ben.hardcastle\.venv\PCbuild\amd64 "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\ATLMFC\lib\x64" "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\lib\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\lib\um\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.18362.0\ucrt\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.18362.0\um\x64" /EXPORT:PyInit_compat_ext build\temp.win-amd64-cpython-38\Release\src/wavpack_numcodecs/compat_ext.obj /OUT:build\lib.win-amd64-cpython-38\wavpack_numcodecs\compat_ext.cp38-win_amd64.pyd /IMPLIB:build\temp.win-amd64-cpython-38\Release\src/wavpack_numcodecs\compat_ext.cp38-win_amd64.lib
         Creating library build\temp.win-amd64-cpython-38\Release\src/wavpack_numcodecs\compat_ext.cp38-win_amd64.lib and object build\temp.win-amd64-cpython-38\Release\src/wavpack_numcodecs\compat_ext.cp38-win_amd64.exp
      Generating code
      Finished generating code
      building 'wavpack_numcodecs.wavpack' extension
      "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\bin\HostX86\x64\cl.exe" /c /nologo /O2 /W3 /GL /DNDEBUG /MD -Isrc/wavpack_numcodecs/include -IC:\Users\ben.hardcastle\.venv\include -I\\allen\programs\mindscope\workgroups\dynamicrouting\ben\envs\win\np_pipeline_qc\include -I\\allen\programs\mindscope\workgroups\dynamicrouting\ben\envs\win\np_pipeline_qc\Include "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\ATLMFC\include" "-IC:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\include" "-IC:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\include\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\ucrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\shared" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\um" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\winrt" "-IC:\Program Files (x86)\Windows Kits\10\include\10.0.18362.0\cppwinrt" /Tcsrc/wavpack_numcodecs/wavpack.c /Fobuild\temp.win-amd64-cpython-38\Release\src/wavpack_numcodecs/wavpack.obj
      wavpack.c
      C:\Users\ben.hardcastle\AppData\Local\Temp\pip-install-xr4e_kat\wavpack-numcodecs_4234e43e7b7b4c058d9781bd3b6fddff\src\wavpack_numcodecs\src/encoder.c(30): warning C4267: '=': conversion from 'size_t' to 'int32_t', possible loss of data
      C:\Users\ben.hardcastle\AppData\Local\Temp\pip-install-xr4e_kat\wavpack-numcodecs_4234e43e7b7b4c058d9781bd3b6fddff\src\wavpack_numcodecs\src/encoder.c(111): warning C4267: '=': conversion from 'size_t' to 'int', possible loss of data
      C:\Users\ben.hardcastle\AppData\Local\Temp\pip-install-xr4e_kat\wavpack-numcodecs_4234e43e7b7b4c058d9781bd3b6fddff\src\wavpack_numcodecs\src/encoder.c(118): warning C4267: '=': conversion from 'size_t' to 'int32_t', possible loss of data
      C:\Users\ben.hardcastle\AppData\Local\Temp\pip-install-xr4e_kat\wavpack-numcodecs_4234e43e7b7b4c058d9781bd3b6fddff\src\wavpack_numcodecs\src/encoder.c(147): warning C4267: 'function': conversion from 'size_t' to 'uint32_t', possible loss of data
      C:\Users\ben.hardcastle\AppData\Local\Temp\pip-install-xr4e_kat\wavpack-numcodecs_4234e43e7b7b4c058d9781bd3b6fddff\src\wavpack_numcodecs\src/encoder.c(163): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
      C:\Users\ben.hardcastle\AppData\Local\Temp\pip-install-xr4e_kat\wavpack-numcodecs_4234e43e7b7b4c058d9781bd3b6fddff\src\wavpack_numcodecs\src/encoder.c(166): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
      C:\Users\ben.hardcastle\AppData\Local\Temp\pip-install-xr4e_kat\wavpack-numcodecs_4234e43e7b7b4c058d9781bd3b6fddff\src\wavpack_numcodecs\src/decoder.c(44): warning C4267: '-=': conversion from 'size_t' to 'int32_t', possible loss of data
      C:\Users\ben.hardcastle\AppData\Local\Temp\pip-install-xr4e_kat\wavpack-numcodecs_4234e43e7b7b4c058d9781bd3b6fddff\src\wavpack_numcodecs\src/decoder.c(106): warning C4005: 'BUFFER_SAMPLES': macro redefinition
      C:\Users\ben.hardcastle\AppData\Local\Temp\pip-install-xr4e_kat\wavpack-numcodecs_4234e43e7b7b4c058d9781bd3b6fddff\src\wavpack_numcodecs\src/encoder.c(47): note: see previous definition of 'BUFFER_SAMPLES'
      C:\Users\ben.hardcastle\AppData\Local\Temp\pip-install-xr4e_kat\wavpack-numcodecs_4234e43e7b7b4c058d9781bd3b6fddff\src\wavpack_numcodecs\src/decoder.c(160): warning C4267: 'initializing': conversion from 'size_t' to 'int', possible loss of data
      src/wavpack_numcodecs/wavpack.c(2197): warning C4267: '=': conversion from 'size_t' to 'unsigned long', possible loss of data
      src/wavpack_numcodecs/wavpack.c(2245): warning C4267: '=': conversion from 'size_t' to 'unsigned long', possible loss of data
      src/wavpack_numcodecs/wavpack.c(2620): warning C4267: '=': conversion from 'size_t' to 'unsigned long', possible loss of data
      src/wavpack_numcodecs/wavpack.c(2776): warning C4267: '=': conversion from 'size_t' to 'unsigned long', possible loss of data
      src/wavpack_numcodecs/wavpack.c(2802): warning C4267: '=': conversion from 'size_t' to 'unsigned long', possible loss of data
      "C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\bin\HostX86\x64\link.exe" /nologo /INCREMENTAL:NO /LTCG /DLL /MANIFEST:EMBED,ID=2 /MANIFESTUAC:NO /LIBPATH:C:\Users\ben.hardcastle\.venv\libs /LIBPATH:\\allen\programs\mindscope\workgroups\dynamicrouting\ben\envs\win\np_pipeline_qc\libs /LIBPATH:\\allen\programs\mindscope\workgroups\dynamicrouting\ben\envs\win\np_pipeline_qc /LIBPATH:C:\Users\ben.hardcastle\.venv\PCbuild\amd64 "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\ATLMFC\lib\x64" "/LIBPATH:C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Tools\MSVC\14.24.28314\lib\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\NETFXSDK\4.8\lib\um\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.18362.0\ucrt\x64" "/LIBPATH:C:\Program Files (x86)\Windows Kits\10\lib\10.0.18362.0\um\x64" wavpackdll.lib /EXPORT:PyInit_wavpack build\temp.win-amd64-cpython-38\Release\src/wavpack_numcodecs/wavpack.obj /OUT:build\lib.win-amd64-cpython-38\wavpack_numcodecs\wavpack.cp38-win_amd64.pyd /IMPLIB:build\temp.win-amd64-cpython-38\Release\src/wavpack_numcodecs\wavpack.cp38-win_amd64.lib /LIBPATH:src/wavpack_numcodecs/libraries/5.7.0/windows-x86_64
      LINK : fatal error LNK1181: cannot open input file 'wavpackdll.lib'
      C:\Users\ben.hardcastle\AppData\Local\Temp\pip-build-env-ndx4tt6e\overlay\Lib\site-packages\setuptools\command\build_py.py:220: _Warning: Package 'wavpack_numcodecs.include.wavpack' is absent from the `packages` configuration.
      !!
```

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,8 +1,0 @@
-include requirements.txt
-recursive-include wavpack_numcodecs *.c
-recursive-include wavpack_numcodecs *.pyx
-recursive-include wavpack_numcodecs *.pxd
-recursive-include wavpack_numcodecs *.lib
-recursive-include wavpack_numcodecs *.dll
-recursive-include wavpack_numcodecs/include *.h
-recursive-include wavpack_numcodecs/libraries *

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -43,7 +43,7 @@ build-backend = "setuptools.build_meta"
 
 [tool.setuptools]
 include-package-data = true
-package-data = {"wavpack_numcodecs" = ["**/*.pyx", "**/*.pxd", "**/*.c", "**/*.h", "**/*.so", "**/*.so.*", "**/*.dll"]}
+package-data = {"wavpack_numcodecs" = ["**/*.pyx", "**/*.pxd", "**/*.c", "**/*.h", "**/*.so", "**/*.so.*", "**/*.dll", "**/*.lib"]}
 
 [tool.setuptools.exclude-package-data]
 wavpack_numcodecs = ["**/tests/test_*"]
@@ -56,3 +56,9 @@ exclude = ["wavpack_numcodecs.*.tests"]
 
 [tool.black]
 line-length = 120
+
+[tool.pytest.ini_options]
+markers = [
+    "numcodecs",
+    "zarr"
+]

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,6 @@
 [project]
 name = "wavpack-numcodecs"
-version = "0.2.1"
+version = "0.2.2"
 description = "Numcodecs implementation of WavPack audio codec."
 authors = [
     {name = "Alessio Buccino", email = "alessiop.buccino@gmail.com"},

--- a/pytest.ini
+++ b/pytest.ini
@@ -1,4 +1,0 @@
-[pytest]
-markers =
-    numcodecs
-    zarr

